winlogbeat/etc/fields.yml

```diff
@@ -228,7 +228,7 @@ eventlog:
       description: The version number of the event's definition.
 
     - name: xml
-      type: string
```

Comment:
Based on @tsg's comment then we should keep your change to `text`.

Most people won't be using this `xml` field. You have to enable it by setting `include_xml: true` and it's mainly for debugging purposes. If you did need it for some reason, I would expect it would go to Logstash, be parsed, then dropped prior to ES indexing.